ignite/metrics/regression/spearman_correlation.py

```diff
@@ -84,7 +88,7 @@ def __init__(
         except ImportError:
             raise ModuleNotFoundError("This module requires scipy to be installed.")
 
-        super().__init__(_get_spearman_r(), output_transform, check_compute_fn, device, skip_unrolling)
+        super().__init__(_get_spearman_r(), output_transform, True, device, skip_unrolling)
```

Comment:
It might be better to add `check_compute_fn` to the docstring than hard-coding the value.
Alternatively, we could replace in the docstring and the constructor all the inherited args with `*args, **kwargs` and say that they are from EpochMetric...